[2025-06-24T10:24:30.341+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.model_training manual__2025-06-24T10:24:20.242271+00:00 [queued]>
[2025-06-24T10:24:30.353+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.model_training manual__2025-06-24T10:24:20.242271+00:00 [queued]>
[2025-06-24T10:24:30.354+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-24T10:24:30.378+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_training> on 2025-06-24 10:24:20.242271+00:00
[2025-06-24T10:24:30.385+0000] {standard_task_runner.py:60} INFO - Started process 307 to run task
[2025-06-24T10:24:30.391+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'model_training', 'manual__2025-06-24T10:24:20.242271+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmpgzsdmay6']
[2025-06-24T10:24:30.393+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask model_training
[2025-06-24T10:24:30.461+0000] {task_command.py:423} INFO - Running <TaskInstance: ml_pipeline.model_training manual__2025-06-24T10:24:20.242271+00:00 [running]> on host 3fe363fae776
[2025-06-24T10:24:30.559+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarah' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='model_training' AIRFLOW_CTX_EXECUTION_DATE='2025-06-24T10:24:20.242271+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-24T10:24:20.242271+00:00'
[2025-06-24T10:25:12.612+0000] {logging_mixin.py:188} INFO - ‚úÖ Subprocess stdout:
[2025-06-24T10:25:12.613+0000] {logging_mixin.py:188} INFO -  Fitting 3 folds for each of 10 candidates, totalling 30 fits
[2025-06-24T10:25:12.614+0000] {logging_mixin.py:188} INFO - ‚úÖ Subprocess stderr:
[2025-06-24T10:25:12.614+0000] {logging_mixin.py:188} INFO -  2025-06-24 10:24:34,545 - training.utils.common - INFO - YAML charg√© : /opt/***/project/config/config.yaml
2025-06-24 10:24:34,619 - training.utils.common - INFO - YAML charg√© : /opt/***/project/config/params.yaml
2025-06-24 10:24:34,637 - training.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/models
2025-06-24 10:24:34,641 - training.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/models
2025-06-24 10:24:37,002 - __main__ - INFO - Loaded processed data: /opt/***/project/data/processed/clean_data.csv (shape=(108060, 13))
2025-06-24 10:24:37,112 - __main__ - INFO - Split data: train_shape=(86448, 10), test_shape=(21612, 10)
2025/06/24 10:24:37 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet

2025-06-24 10:24:37,771 - __main__ - INFO - Starting hyperparameter search and training.
2025-06-24 10:24:54,029 - __main__ - INFO - Best params: {'colsample_bytree': 0.6603626777682708, 'gamma': 0.01657240506474713, 'learning_rate': 0.1464270668036927, 'max_depth': 6, 'n_estimators': 113, 'subsample': 0.8049969668130992}
2025-06-24 10:24:54,030 - __main__ - INFO - R^2 score: 0.9942
Registered model 'PriceFixerModel' already exists. Creating a new version of this model...
2025/06/24 10:25:03 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: PriceFixerModel, version 20
Created version '20' of model 'PriceFixerModel'.
2025/06/24 10:25:09 INFO mlflow.tracking._tracking_service.client: üèÉ View run XGBoost_RandSearch at: https://dagshub.com/Gsasana/rakuten-mlops.mlflow/#/experiments/1/runs/4bcd942db4844401ba1cd4f6c2febf55.
2025/06/24 10:25:09 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: https://dagshub.com/Gsasana/rakuten-mlops.mlflow/#/experiments/1.
2025-06-24 10:25:11,824 - training.repository.repository - INFO - Mod√®le sauvegard√© ‚Üí /opt/***/project/models/xgb_model.pkl
2025-06-24 10:25:11,824 - __main__ - INFO - Model saved locally at: /opt/***/project/models/xgb_model.pkl
[2025-06-24T10:25:12.615+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-24T10:25:12.625+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=model_training, execution_date=20250624T102420, start_date=20250624T102430, end_date=20250624T102512
[2025-06-24T10:25:12.662+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-24T10:25:12.683+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
